requirements.txt

```diff
@@ -6,3 +6,4 @@ djangorestframework>=3.11,<3.12
 mac-vendor-lookup~=0.1
 swapper~=1.1
 django-cache-memoize~=0.1
+elasticsearch-dsl>=7.0.0,<8.0.0
```

Comment:
we should make both this and the influxdb client lib optional and move these to the extra_require in setup.py, see https://github.com/openwisp/openwisp-utils/blob/master/setup.py#L49-L60
we should have `inflxudb` and `elasticsearch`, is a cleaner solution.
In the build you can install everything with `pip install -e .[influxdb,elasticsearch]`